### ISSUE TITLE
Unable to scroll up with keyboard 

### ISSUE THREAD

**Issue by nus-se-script** [nus-cs2103-AY2425S1/pe-dev-response#3217]
Since this is optimised for keyboard users and fast typers, being at the end of a really long list means that I cannot scroll up without using my mouse. Some planned enhancements can include better CLI commands such that I can scroll with the keyboard instead, benefitting more as a fast typer.

<hr>
<sub>[original: nus-cs2103-AY2425S1/pe-interim#2002] [original labels: severity.Medium type.FeatureFlaw]</sub>

**Comment by maertan** [comment]
# Team's Response
It is possible to scroll using keyboard, so this is not a feature flaw.

Steps to do this:
1. Press `tab` until one of the contacts is highlighted.
2. Use the up and down arrows to navigate up and down the contacts list.




## Duplicate status (if any):
--
